Gemfile

```diff
@@ -97,7 +97,7 @@ gem 'rgeo-geojson'
 gem 'ruby-saml'
 gem 'rubyzip', '>= 1.3.0'
 gem 'savon'
-gem 'sentry-raven', '2.9.0' # don't change gem version unless sentry server is also upgraded
+gem 'sentry-raven'
```

Comment:
You might consider locking to < 3.0 or to ~>2.13 so you make it clear you can't/won't go to 3.0 if there's a reason preventing it.